var/spack/repos/builtin/packages/spatialdata/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Spatialdata(AutotoolsPackage):
+    """Spatialdata is a C++ library for
+
+    interpolating values for spatially distributed data,
+    converting coordinates among geographic projections using Proj,
+    nondimensionalization of quantities,
+    specification of units via Pyre (optional).
+
+    This library is used in the finite-element code PyLith
+    (https://github.com/geodynamics/pylith). The primary focus is specification
+    of parameters that vary in space, such as values for boundary conditions
+    and parameters of constitutive models. This provides a specification of
+    these parameters independent of the discretization."""
```

Comment:
Can you edit this so that there is a <100 character single-line summary on the first line, and the rest follows?